### DISCUSSION TITLE
How to change laravel version after project created?

### DISCUSSION THREAD

**Discussion by jay2success** [TiagoSilvaPereira/vemto-issues#342] [Q&A] [answered]
Hello,

In project settings, the laravel version select field is disabled.
How i can change laravel version after project created?
Thanks.

**Comment by TiagoSilvaPereira** [comment, marked as answer]
@jay2success it is currently not possible directly from Vemto. Still, you can do it by editing the project JSON file under the Vemto storage folder, or by exporting the project to .vemto, editing the .vemto file then importing it again.

But be careful, if you change the project Laravel version it is recommended to regenerate the project code, as Vemto templates may differ for different Laravel versions

**Reply by jay2success** [reply]
Thanks a lot for your reply
